apps/voting-aggregator/contracts/VotingAggregator.sol

```diff
@@ -66,6 +67,7 @@ contract VotingAggregator is IERC20WithCheckpointing, IForwarder, IsContract, ER
     string public symbol;
     uint8 public decimals;
 
+    mapping (address => bool) public powerSourceAdded;
```

Comment:
I wonder if we could map to the id, so we could allow accessing power sources by address instead of by this artificial id.

Reply:
Yes! I think this is much better indeed :). It may also make it easier for us to implement source removals if we ever need to.